README.md

```diff
@@ -25,3 +25,97 @@ Check our [Documentation](/docs) to start using our Kubernetes operator.
 ## License
 
 Please check the [LICENSE](LICENSE) file
+
```

Comment:
This looks great! Great job @as51340. 🚀 

Since this huge PR sets a lot of basic groundwork, can you expand this README to include more information on this? I can see you already planned to do this.

Reply:
proposals what should I include?
I planed to add add just installation, I am following what folks at Mongo were doing.

Reply:
I would add the following things: 

- How it is connected/unconnected to the rest of k8s stuff in Memgraph (primarily helm charts) 
- Enterprise notes
- Supported features 

What do you think? Should we create documents at this point, or should they be part of the next release? 

If we are not creating them now, maybe we could just open an issue and link it to PR.

Reply:
I would merge this as it is and then we can easily add more docs separately